.changeset/tough-cows-rule.md

```diff
@@ -0,0 +1,8 @@
+---
+"@hashicorp/design-system-tokens": major
+---
+
+MODES - TODOs
+
+- Decide if this is a `major` or a `minor`
```

Comment:
Is it expected to cause breaking changes? If the main result from the consumer standpoint is that themes are added but the default styles/theme stays the same unless they explicitly opt in to using a different theme then we could consider that as a "minor" I think.